src/test_localization_landmark_update.cpp

```diff
@@ -90,12 +90,15 @@ TEST(LocalizationLandmarkUpdate, Jacobian) {
     update.evalInnovationShort(y, state);
     EXPECT_TRUE(y.pix().isApprox(-kKeypointDisturbance));
 
+    static constexpr unsigned int kDimState = State::D_;
```

Comment:
this is the actual compilation fix. Without these intermediate variables, this code would result in a linker error.